src/sidebar/store/modules/drafts.js

```diff
@@ -96,14 +97,41 @@ function createDraft(annotation, changes) {
   };
 }
 
-/** Remove all drafts. */
+/**
+ * Remove any drafts that are empty.
+ *
+ * An empty draft has no text and no reference tags.
+ */
+
+function deleteNewAndEmptyDrafts() {
+  const annotations = require('./annotations');
```

Comment:
This one is beyond me, It won't work imported from the top of the file, I could use some help figuring out why. The other stores import just fine.

Reply:
There is a circular import here where the "drafts" module requires the "annotations" module and vice-versa. CommonJS imports don't support circular imports properly. If we converted this to ES imports that would work. Circular imports are often something to be wary of as it indicates coupling which might be problematic in future. In this context, I think there are a few ways this could be done

1. Refactor the logic so that either 1) drafts knows about annotations OR 2) annotations knows about drafts, but not both.
2. Extract this action into a separate module which knows about annotations and drafts
3. Combine the modules together so there is no circular import to worry about

I think I'd prefer to merge this as-is though so we can move on.